controller/space.go

```diff
@@ -318,13 +318,21 @@ func deleteCodebases(
 		path = client.DeleteCodebasePath(*cb.ID)
 		resp, err := cl.DeleteCodebase(ctx, path)
 		if err != nil {
+			log.Error(ctx, map[string]interface{}{
+				"error":    err,
+				"response": resp,
+			}, "failed to delete codebase %s", cb.ID)
 			errorsList = append(errorsList,
 				errs.Wrapf(err, "could not delete codebase %s", cb.ID))
 			continue
 		}
 		if 200 < resp.StatusCode && resp.StatusCode >= 300 {
 			formattedErrors, err := cl.DecodeJSONAPIErrors(resp)
 			if err != nil {
+				log.Error(ctx, map[string]interface{}{
+					"error":    err,
```

Comment:
I think in other places we log errors under the key `err` and not `error`. Please check that so we're consistent here and can properly filter in Kibana.

Reply:
@kwk We're using both `err` and `error` https://github.com/fabric8-services/fabric8-wit/blob/31004c1d67b98e0105fef54a04f6c586637a221d/controller/space.go

I can rename it to `err` but it won't add much value. Can we please keep it as it is?

Reply:
We have about 250 log statements according to:

```bash
$ grep -F "log.Error(ctx, map[string]interface{}{" . -r | wc -l
269
```

169 of those use `err` and only 5 use `error`.

```bash
$ grep -F "log.Error(ctx, map[string]interface{}{" . -r --after=2 | grep -F '"err"' | wc -l
169
```

```bash
grep -F "log.Error(ctx, map[string]interface{}{" . -r --after=2 | grep -F '"error"' | wc -l
5
```

So I think you should go with `err`. The value added here is that you can search for this field `err` in Kibana.

Reply:
Thanks @kwk . I have updated it in https://github.com/fabric8-services/fabric8-wit/pull/2310/commits/7928b20b90bcf7a17f447e7f8e42519fcb054daa